### PR TITLE
Ft login phantom

### PR DESCRIPTION
**Pull request by NyiingangoPacifique**
#### What does this PR do?

- Login to Phantom App

#### Description of Task to be completed?
- User must be able to login to phantom
- Appropriate error messages are displayed on unsuccessful attempts
- Appropriate error messages are displayed on invalid inputs (Form validation).
#### How should this be manually tested?
-first pull branch `git pull ft-login-to-phantom`
-install dependencies `npm install`
-then run` npm run dev`
#### Screenshots
Screenshot from 2022-04-20 15-14-11

### Responsive
Screenshot from 2022-04-20 15-18-03

### Displaying error from server
Screenshot from 2022-04-20 15-14-34

### CODE REVIEW

File: src/container/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+      password,
+    }
+    dispatch(login(userData))
+      
+    return setValidation(errors);
+
+  }
+  // if (isLoading) {
+  //   return <Spinner />
+  // }
```
Comment (SimonImanigirimpuhwe) [inline src/container/Login.jsx]: Remove this commented code

File: src/container/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    setValues({ ...values, showPasswordd: !values.showPasswordd });
+  };
+  
+  const handleMouseDownPassword = (event) => {
+    event.preventDefault();
+  };
+  
+  const handleChange = (prop) => (event) => {
+    setValues({ ...values, [prop]: event.target.value });
+  };
```
Comment (SimonImanigirimpuhwe) [inline src/container/Login.jsx]: When there is an `error message` and user starts typing for retry, the `error message` should be removed from the UI. 
To achieve that you need to reset `error message` here

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]

File: src/container/Login.jsx (hunk 1 above)
Comment (NyiingangoPacifique) [reply src/container/Login.jsx]: okay

File: src/container/Login.jsx (hunk 2 above)
Comment (NyiingangoPacifique) [reply src/container/Login.jsx]: on it

**Pull request closed without merging**